isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -4322,20 +4360,30 @@ namespace isobus
 		return retVal;
 	}
 
-	bool VirtualTerminalClient::send_command(const std::vector<std::uint8_t> &data, CANIdentifier::CANPriority priority) const
+	bool VirtualTerminalClient::send_command(const std::vector<std::uint8_t> &data, CANIdentifier::CANPriority priority)
 	{
+		if (commandAwaitingResponse)
```

Comment:
We should probably add a timeout and check that as well in here? If we're waiting more than 1.5s (I think that's what the standard said?) then we could process the transmit as normal and log what happened

Reply:
Oh yeah of course, totally forgot it